api/src/main/java/org/openmrs/module/pihmalawi/task/MigrateEIDTestResultsTask.java

```diff
@@ -0,0 +1,87 @@
+package org.openmrs.module.pihmalawi.task;
+
+import org.openmrs.Concept;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterType;
+import org.openmrs.Obs;
+import org.openmrs.api.context.Context;
+import org.openmrs.scheduler.tasks.AbstractTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+// TODO remove this after EIDTestResult migration is complete (will need to remove task as well)
+public class MigrateEIDTestResultsTask extends AbstractTask {
+
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    @Override
+    public void execute() {
+
+        EncounterType eidScreening = Context.getEncounterService().getEncounterTypeByUuid("8383DE35-5145-4953-A018-34876B797F3E");  // EID Screening
+        EncounterType exposedChildInitial = Context.getEncounterService().getEncounterTypeByUuid("664bcbb0-977f-11e1-8993-905e29aff6c1");  // Exposed Child Initial
+        Concept eidTestConstruct = Context.getConceptService().getConceptByMapping("2168", "PIH Malawi");  // HIV Test Construct
+
+        Concept sampleDateConcept = Context.getConceptService().getConceptByMapping("6108", "PIH Malawi");    // Date of blood sample
+       // Concept resultDateConcept = Context.getConceptService().getConceptByMapping("6110", "PIH Malawi");   // Date of result concept
```

Comment:
This is excluded intentionally?

Reply:
Yeah, I added some functionality so that if it doesn't find a matching sample date obs it searches for a result date obs and uses that instead if found... not sure if we really want to use that though, so I commented that part out until we decide.